drydock/templates/kustomized/global-stack/kustomization.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: {{ DRYDOCK_GLOBAL_STACK_NAMESPACE }}
+
+{%- if DRYDOCK_GLOBAL_RESOURCES|default(false) %}
+resources:
+  {{ patch("drydock-global-resources") }}
+{%- endif %}
+
+helmCharts:
+- name: kube-prometheus-stack
+  repo: https://prometheus-community.github.io/helm-charts
+  version: {{ DRYDOCK_PROMETHEUS_VERSION }}
+  releaseName: prometheus
+  includeCRDs: true
+{%- if DRYDOCK_PROMETHEUS_VALUES | default(false) %}
```

Comment:
I'm thinking of not using default here and instead having them in the defaults.yml file? That way we can refer to that file to know what variables are available.

Reply:
Done! :)